README.md

```diff
@@ -7,11 +7,23 @@ This is KubeVirt API an add-on for Kubernetes.
 This SDK is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
 - API version: 1.0.0
-- Package version: 1.0.0
+- Package version: 0.0.1
 - Generator version: 7.9.0
 - Build package: org.openapitools.codegen.languages.RubyClientCodegen
 For more information, please visit [https://github.com/kubevirt/kubevirt](https://github.com/kubevirt/kubevirt)
 
+## Generation
+
+To autogenerate the gem from the openapi specification you must first download the openapi-generator-cli from https://openapi-generator.tech/docs/installation/
+
+For this example we'll be using the `openapi-generator-cli.jar`.
+
+```bash
+java -jar /var/lib/openapi/7.9.0/openapi-generator-cli.jar generate \
+ --input-spec https://raw.githubusercontent.com/kubevirt/kubevirt/refs/heads/main/api/openapi-spec/swagger.json \
+ --skip-validate-spec --generator-name ruby --config .openapi-config.json
```

Comment:
NOTE the --skip-validate-spec is needed here because the kubevirt swagger.json doesn't pass spec validation.